src/io.jl

```diff
@@ -84,9 +84,16 @@ Use `compress` to specify the compression with the following options:
 - `:bz2`: BZ2
 - `:xz`: XZ
 - `:none`: no compression
+
+The parameter `unit_vsize` specifies the size (in bytes) of an intermediate
+buffer that is used to speed up the writing.
+
+You can also directly provide a preallocated buffer as a `Vector`.
+In that case, `unit_vsize` has no effect.
+Note that `eltype(buffer)` must match the data type of the MRC data.
 """
 write(::Any, ::MRCData)
-function Base.write(io::IO, d::MRCData; compress=:none, unit_vsize=4096)
+function Base.write(io::IO, d::MRCData; compress=:none, unit_vsize=4096, buffer::Union{Nothing, Vector}=nothing)
```

Comment:
Now that we're documenting the `unit_vsize` and making it part of the API, maybe it's a good time to pick a more intuitive name, e.g. `buffer_size`. What do you think?

Reply:
Good point. I think `buffer_size` is more intuitive. However, it loses the connection to the reason behind all this, namely the faster vectorized (which is what I assume the `v` in `unit_vsize` stands for) writes. This could be mentioned in the documentation, though, I guess.